doc/connectivity/networking/native_sim_setup.rst

```diff
@@ -142,3 +142,18 @@ For TCP test, type:
 .. code-block:: console
 
    ./echo-client -t 127.0.0.1
+
+Setting interface name from command line
+****************************************
+
+By default the ethernet interface name used by native_sim is determined by
+:kconfig:option:`CONFIG_ETH_NATIVE_POSIX_DRV_NAME`. There is option to choose
+the interface on command line.
+
+First you have to enable :kconfig:option:`CONFIG_ETH_NATIVE_POSIX_DRV_NAME_FROM_CMD_LINE`.
```

Comment:
No need to require this. Let's just do so that if the user does not provide anything from cmd line we use the default from kconfig, and otherwise what the user provided